doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -169,3 +169,47 @@ Currently, the following intents are defined within the ``IntentEnum`` enumerato
   * ``kEndUserSupport`` - Logs created by the product maker to be used for end-user support.
   * ``kNetworkDiag``- Logs created by the network layer to be used for network diagnostic.
   * ``kCrashLogs`` - Logs created during a device crash, to be obtained from the node.
+
+Crash logs
+----------
+
+The crash logs module is a part of the diagnostic log module and contains the data of the most recent device crash.
+When a crash occurs, the device saves the crash data to the defined retained RAM partition.
+This means that the :ref:`ug_matter_diagnostic_logs_snippet` must be added to the build to enable crash log support.
```

Comment:
```suggestion
Because it uses the retained RAM partition, the :ref:`ug_matter_diagnostic_logs_snippet` must be added to the build to enable crash log support.
```